map/map_loader/README.md

```diff
@@ -38,13 +38,14 @@ Please see [the description of `GetDifferentialPointCloudMap.srv`](https://githu
+| :---------------------------- | :------ | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------------ |
+| enable_whole_load             | bool    | A flag to enable raw pointcloud map publishing                                                                                                                                                                                                                                                                                      | true          |
+| enable_downsampled_whole_load | bool    | A flag to enable downsampled pointcloud map publishing                                                                                                                                                                                                                                                                              | false         |
+| enable_partial_load           | bool    | A flag to enable partial pointcloud map server                                                                                                                                                                                                                                                                                      | false         |
+| enable_differential_load      | bool    | A flag to enable differential pointcloud map server                                                                                                                                                                                                                                                                                 | false         |
+| leaf_size                     | float   | Downsampling leaf size (only used when enable_downsampled_whole_load is set true)                                                                                                                                                                                                                                                   | 3.0           |
+| map_upper_limit               | int64_t | Limit data size of topic/service/action (determined by cyclonedds) (only used when differential_load enabled). Default value is calculated as follows; <br> map_upper_limit = floor((maximum receive buffer size)\* 0.8) <br> If maximum receive buffer size in your PC's environment is 2147483647, map_upper_limit is 1717986917. | 1717986917    |
```

Comment:
How about clarifying where this magic number come from? (probably here? https://github.com/eclipse-cyclonedds/cyclonedds/blob/09a02f5b2b9ba856d7d4c616510252cf9e508ec2/docs/manual/config/config_file_reference.rst#cycloneddsdomaininternalmaxsamplesize)

Reply:
Thank you!!
I added description in https://github.com/autowarefoundation/autoware.universe/pull/3162/commits/56352bbc57b84f00fdcba9fd14144d12812da67d